gtda/sampling/sample_point_clouds.py

```diff
@@ -0,0 +1,177 @@
+"""Point-cloud–related plotting functions and classes."""
+# License: GNU AGPLv3
+
+import numpy as np
+from scipy.optimize import root_scalar
+
+from gtda.utils import validate_params
+from gtda.utils.intervals import Interval
```

Comment:
In the other script you are importing Interval as Int. There is no consistency. And we are in the same PR.